boxes.cc

```diff
@@ -31,14 +34,23 @@ using namespace benchmark;
 // Boxes:
 // Spawn a single box and record accuracy for momentum and enery
 // conservation
-void BoxesTest::Boxes(const std::string &_physicsEngine
-                    , double _dt
-                    , int _modelCount
-                    , bool _collision
-                    , bool _complex)
+void BoxesTest::Boxes(const std::string& _physicsEngine, double _dt, int _modelCount, bool _collision, bool _complex)
 {
+  ASSERT_GT(_modelCount, 0);
+
+  std::stringstream command;
+  command << "erb "
+          << "engine=" << _physicsEngine << " model_count=" << _modelCount << " collision=" << _collision
+          << " complex=" << _complex << " dt=" << _dt << " " << WORLDS_FILE_PATH << "/boxes.world.erb > "
+          << WORLDS_FILE_PATH << "/boxes.world";
```

Comment:
we could consider encoding the parameters in the name of the generated world:

~~~
boxes_collision1_complex0_dt1e-3_modelCount1000.world
boxes_collision0_complex1_dt1e-2_modelCount10.world
~~~

then one could share the world and an output log file to document their results

Reply:
yes, this would be nice, I have made the suggested change could please review it again at your convenience?